src/Makefile

```diff
@@ -75,12 +75,16 @@ $(CLEANDIRS):
 	$(MAKE) -s -C $(@:clean-%=%) clean
 
 phpvendors:
-	composer install -q
+	$(BUILDDIRS)/composer install -q
```

Comment:
`$BUILDDIRS` is a **list** defined at line `38` not a single directory? Does this work?